tests/nemo_text_processing/en/data_text_normalization/test_cases_word.txt

```diff
@@ -35,3 +35,6 @@ $ and 5% or %~dollar and five percent or percent sign
 love him while we may,~love him while we may,
 mar~mar
 /$€₩£BB¥#%AA and $€₩£¥#%~slash dollar euro won pound BB yen hash percent sign AA and dollar euro won pound yen hash percent sign
+1!hello~one ! hello
+love him while we may,~love him while we may ,
```

Comment:
no space before comma at end:
correct test case:
love him while we may,~love him while we may,